lib/network/call_account_api.js

```diff
@@ -1,134 +1,22 @@
 // eslint-disable-next-line import/order
 const config = require("../config");
-const https = /^http:/.test(config().upload_prefix) ? require('http') : require('https');
-const querystring = require("querystring");
-const Q = require('q');
-const url = require('url');
```

Comment:
@patrick-tolosa , why were those removed? just clean up that is not related to the PR?

Reply:
Got it... extracted.